atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -870,7 +873,8 @@ private Transaction getReadOnlyTransaction(final long commitTs) {
                 transactionConfig,
                 conflictTracer,
                 tableLevelMetricsController,
-                knowledge) {
+                knowledge,
+                transactionalExpectationsConfigReference.get()) {
```

Comment:
It's not entirely clear how we should treat this, most likely we want to have a no expectations config for this as we only run it for serializable checks